src/System Application/App/Document Processor/app.json

```diff
@@ -0,0 +1,37 @@
+{
+  "id": "2c7afcfb-9625-4ace-b972-5889d045ce8f",
+  "name": "Document Processor",
```

Comment:
Consider naming PDF Document if we only want to limit this module to PDF documents. I think it is valid, we could introduce a module for each of the specific documents we process, like Word, PDF...

Maybe we do not need to limit the module to extracting the data only, but to PDF documents.

If we want the module that extracts the data from all different documents it should be renamed to Document Data Extractor or something like this. 

PDF Document is better for me as it would be easier to use.

Reply:
It does not have a clear purpose. Document Processor is incredibly generic term.